### DISCUSSION TITLE
Wheelpicker / UIPicker in Quasar app

### DISCUSSION THREAD

**Discussion by i-mage** [quasarframework/quasar#9546] [General - Components / Directives / etc] [answered]
I would like to ask you whether the Wheelpicker / UIPicker-like component is available for Quasar apps. 

UIPicker examples: https://www.google.com/search?q=uipicker&tbm=isch&ved=2ahUKEwjux_mHu_zwAhWCq6QKHTU0C70Q2-cCegQIABAA&oq=uipicker&gs_lcp=CgNpbWcQAzICCAAyAggAMgIIADICCAAyAggAMgIIADIECAAQHjIGCAAQChAYMgQIABAYMgQIABAYUIAIWIAIYKAJaABwAHgAgAFIiAFIkgEBMZgBAKABAaoBC2d3cy13aXotaW1nwAEB&sclient=img&ei=ylC5YO46gteSBbXorOgL&bih=594&biw=1068

**Comment by ontwikkelfabriek** [comment, marked as answer]
@i-mage 

Yes it's available as a Quasar extension:
https://quasar.dev/app-extensions/discover#official-app-extensions
https://github.com/quasarframework/quasar-ui-qscroller

Demo:
https://quasarframework.github.io/quasar-ui-qscroller/demo

**Reply by i-mage** [reply]
That's awesome! Thank you.
